library/core/src/result.rs

```diff
@@ -1,9 +1,9 @@
 //! Error handling with the `Result` type.
 //!
 //! [`Result<T, E>`][`Result`] is the type used for returning and propagating
-//! errors. It is an enum with the variants, [`Ok(T)`], representing
-//! success and containing a value, and [`Err(E)`], representing error
-//! and containing an error value.
+//! errors. It is an enum with the variants, [`Ok(T)`], representing success and
```

Comment:
This change is unrelated. You've changed the formatting all over the place in this patch, can you make it so that the only change is the new addition?

Reply:
The formatting change is already split to a separate commit from the new content, doesn't seem like those changes justify their own PR (separation of concerns vs. review resource batching...)

Reply:
I just used my editor to rewrap the comments while editing the new paragraph, and it also rewrapped these other paragraphs. I didn't think it mattered since the generated rustdoc is the same, but I can adjust it if people think it matters.